### PR TITLE
Update number.md

### PR DESCRIPTION
**Pull request by heynemann**
The docs are apparently wrong when comparing to the string ones. Hope it is correct. If any other changes are required, please tell me and I'll be more than happy to change.

### CODE REVIEW

**Comment by heynemann** [comment]
Sorry for the many commits... As github allows you to squash when merging I didn't squash myself. I can if it is preferred, though.

File: doc/number.md [hunk 1]
```diff
@@ -24,8 +24,9 @@ Creates a new instance of a `Napi::Number` object.
 Napi::Number(napi_env env, napi_value value);
 ```
 
- - `[in] env`: The `napi_env` environment in which to construct the `Napi::Nuber` object.
- - `[in] value`: The `napi_value` which is a handle for a JavaScript `Number`.
+ - `[in] env`: The `napi_env` environment in which to construct the `Napi::Number` object.
+ - `[in] value`: The primitive to wrap.
```
Comment (gabrielschulhof) [inline doc/number.md]: IMO this explanation is too brief. How about "The JavaScript value holding a number."?

File: doc/number.md [hunk 2]
```diff
@@ -36,8 +37,8 @@ Napi::Number(napi_env env, napi_value value);
 ```cpp
 Napi::Number Napi::Number::New(napi_env env, double value);
 ```
- - `[in] env`: The `napi_env` environment in which to construct the `Napi::Nuber` object.
- - `[in] value`: The `napi_value` which is a handle for a JavaScript `Number`.
+ - `[in] env`: The `napi_env` environment in which to construct the `Napi::Number` object.
+ - `[in] value`: The C++ primitive from which to instantiate the `Napi::Value`. `value` must be a `c++ double`.
```
Comment (gabrielschulhof) [inline doc/number.md]: IMO the explanation here should be "The value from which to instantiate the &#96;Napi::Number&#96;".

Comment (gabrielschulhof) [reply doc/number.md]: The fact that the value is a `double` should be obvious from the prototype.

File: doc/number.md [hunk 3]
```diff
@@ -36,8 +37,8 @@ Napi::Number(napi_env env, napi_value value);
 ```cpp
 Napi::Number Napi::Number::New(napi_env env, double value);
 ```
- - `[in] env`: The `napi_env` environment in which to construct the `Napi::Nuber` object.
- - `[in] value`: The `napi_value` which is a handle for a JavaScript `Number`.
+ - `[in] env`: The `napi_env` environment in which to construct the `Napi::Number` object.
+ - `[in] value`: The C++ primitive from which to instantiate the `Napi::Value`.
```
Comment (gabrielschulhof) [inline doc/number.md]: "... from which to instantiate the &#96;Napi::Number&#96;", right, because the return value is `Napi::Number`?

Comment (heynemann) [reply doc/number.md]: In the string page it says Napi::Value, but I changed it. Maybe we need to edit the string docs as well?

Comment (gabrielschulhof) [reply doc/number.md]: @heynemann we may indeed need to edit the string docs as well.

Comment (gabrielschulhof) [reply doc/number.md]: However, it's OK if we do that in a second PR.

Comment (heynemann) [reply doc/number.md]: Any additional feedback for this PR?

**Comment by mhdawson** [comment]
@heynemann, thanks for the update landed as: 7b87e0b9993942d4404c0a2a209ee28423790914

**Pull request closed without merging**